docs/src/manifolds/heisenbergmatrices.md

```diff
@@ -0,0 +1,7 @@
+# Heisenberg matrices
```

Comment:
we are not fully unified here, but I think calling the file just `Heisenberg.md` would also be fine.

Reply:
OK, done.